src/03-dice-roller/DiceRoller/DiceRoller/DiceRollCommandParser.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace DiceRoller
+{
+    public class DiceRollCommandParser
+    {
+        public List<Dice> ParseDices(string[] inputArray)
```

Comment:
Interesting that you are not giving the full string, but an array of something - yet you check here whether the array contains /roll. If you have already done some magic on the input why not validate it before calling this method? Currently it seems that logic is in two separate places. Either pass in array of dice descriptions (1d6, 1d8 etc) or full command...

Reply:
good point, will do some changes